params/protocol_params.go

```diff
@@ -183,13 +183,13 @@ const (
 	MaxBlobGasPerBlock          = 6 * BlobTxBlobGasPerBlob // Maximum consumable blob gas for data blobs per block
 
 	// BorDefaultMinerGasPrice defines the minimum gas price for bor validators to mine a transaction.
-	BorDefaultMinerGasPrice = 30 * GWei
+	BorDefaultMinerGasPrice = 25 * GWei
```

Comment:
No need to check for Amoy here?

Reply:
No amoy check needed here as the places where these values are used are behind amoy checks.